tests/e2e/data-importers/pg/11-assert.yaml

```diff
@@ -1,9 +1,10 @@
 apiVersion: kuttl.dev/v1
 kind: TestAssert
-timeout: 300
+timeout: 600
 collectors:
   - command: kubectl get db/test-pg-cluster -n ${NAMESPACE} -o yaml
   - command: kubectl get perconapgcluster/test-pg-cluster -n ${NAMESPACE} -o yaml
+  - command: kubectl describe perconapgcluster/test-pg-cluster -n ${NAMESPACE}
```

Comment:
do we need to have both `get perconapgcluster/test-pg-cluster` and `describe perconapgcluster/test-pg-cluster`?